src/components/Search/RenderInput/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { AutocompleteRenderInputParams } from '@mui/material/Autocomplete';
+import TextField from '@mui/material/TextField';
+import './style.scss';
+
+function RenderInput(props: AutocompleteRenderInputParams) {
```

Comment:
previous comment about it was wrong @kozlova-anzhelika . This is component itself, so `Input` or `SearchInput` was right

Reply:
it was not my comment :)

Reply:
@yuliabarysenka  Oops It was mine. I'd like to clarify my thoughts about it. :)
I reckon that this function is a callback of autocomplete component, and used as a method to render custom input.
That's why I proposed to name it as a method using a verb :)
Sorry for confusing !!